src/repository.h

```diff
@@ -33,6 +33,12 @@
 #include <git2.h>
 #include "types.h"
 
+typedef enum {
+	GIT_REFERENCES_ALL = 0,
+	GIT_REFERENCES_BRANCHES = 1,
+	GIT_REFERENCES_TAGS = 2,
+} git_reference_iterator_return_t;
```

Comment:
Linking with my previous comment, probably these should be exposed to Python with `ADD_CONSTANT_INT` and used both by `Repository.references.iterator(...)` and by the `Repository_references_iterator_next` function.